russian/lorem/read.md

```diff
@@ -0,0 +1,12 @@
+На вкус как зефир. Но удивительно, вы можете есть его сколько угодно, даже всю стену. 
+В какой-то момент вам это надоедает, а зефир все не кончается.
+А потом приходят они.
+
+Ваши действия:
+
+[Вызвать милицию](police/call-police.md)
+
+[Позавтракать с Винипухом и Пяточком](breakfast/breakfast-marshmallows.md)
+
+[Спросить о будущем планеты](yes/yesdrugs.md)
+[Скажете "нет" наркотикам](no/nodrugs.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.